boa/src/exec/mod.rs

```diff
@@ -165,6 +165,13 @@ impl Executor for Interpreter {
                 }
                 Ok(result)
             }
+            Node::DoWhileLoop(ref expr, ref cond) => {
+                let mut result = self.run(expr)?;
+                while self.run(cond)?.borrow().is_true() {
+                    result = self.run(expr)?;
+                }
+                Ok(result)
+            }
```

Comment:
Maybe consider changing `expr` to `body`. Just so it's easier to read the code.
```suggestion
            Node::DoWhileLoop(ref body, ref cond) => {
                let mut result = self.run(body)?;
                while self.run(cond)?.borrow().is_true() {
                    result = self.run(body)?;
                }
                Ok(result)
            }
```